ui-tests/src/test/java/io/syndesis/qe/pages/customizations/CustomizationsPage.java

```diff
@@ -0,0 +1,52 @@
+package io.syndesis.qe.pages.customizations;
+
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.$;
+
+import org.openqa.selenium.By;
+
+import com.codeborne.selenide.SelenideElement;
+
+import io.syndesis.qe.pages.SyndesisPageObject;
+import io.syndesis.qe.pages.customizations.extensions.TechExtensionsListComponent;
+import lombok.Getter;
+
+@Getter
+public class CustomizationsPage extends SyndesisPageObject {
+
+	private static final class Element {
+		public static final By ROOT = By.cssSelector("syndesis-customizations");
+	}
+
+	private static final class Link {
+		public static final By TECH_EXTENSIONS = By.linkText("Technical Extensions");
+		public static final By API_CLIENT_CONNECTORS = By.linkText("Technical Extensions");
+	}
+
+	private TechExtensionsListComponent techExtensionsListComponent = new TechExtensionsListComponent();
+
+	@Override
+	public SelenideElement getRootElement() {
+		return $(Element.ROOT).shouldBe(visible);
+	}
+
+	@Override
+	public boolean validate() {
+		try {
+			SelenideElement rootElement = this.getRootElement();
+			rootElement.find(Link.TECH_EXTENSIONS).shouldBe(visible);
+			rootElement.find(Link.API_CLIENT_CONNECTORS).shouldBe(visible);
+		} catch (Exception e) {
```

Comment:
I've been wondering. What exception are you trying to catch here exactly?

Reply:
It seems like abandoned work in progress :-)

Reply:
OK:) Just comment it with TODO then.